extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/ChunkInputStreamGenerator.java

```diff
@@ -3,13 +3,13 @@
 //
 package io.deephaven.extensions.barrage.chunk;
 
+import io.deephaven.chunk.Chunk;
 import io.deephaven.chunk.attributes.Values;
 import io.deephaven.engine.rowset.RowSet;
 import io.deephaven.extensions.barrage.util.DefensiveDrainable;
 import io.deephaven.extensions.barrage.util.StreamReaderOptions;
 import io.deephaven.util.QueryConstants;
 import io.deephaven.util.datastructures.LongSizedDataStructure;
-import io.deephaven.chunk.Chunk;
 import io.deephaven.chunk.ChunkType;
```

Comment:
This is going to get fixed when https://github.com/deephaven/deephaven-core/pull/5889 merges, so that we're not mixing in server changes here - this failed due to "out of disk space" repeatedly last week, but should be ready to go when GHA stops doing that...